internal/exec/runner.go

```diff
@@ -498,7 +498,7 @@ func (r *runner) All(args []string, disableFormat, disableLint, fixFlag bool) er
 	return nil
 }
 
-func (r *runner) GRPC(args, headers []string, address, method, data, callTimeout, connectTimeout, keepaliveTime string, stdin bool) error {
+func (r *runner) GRPC(args, headers []string, address, method, data, callTimeout, connectTimeout, keepaliveTime string, stdin bool, tls bool, insecure bool, cacert string, cert string, key string, serverName string) error {
```

Comment:
See discussion on `--tls` flag. We would need to verify that flags are all correctly set, not just ignore them.

Reply:
Ok I agree with your prior comment. I think producing an error when the tls is not set yet other parameters are specified.